airbyte-integrations/connectors/source-postgres/gradle.properties

```diff
@@ -1 +1,3 @@
-testExecutionConcurrency=-1
\ No newline at end of file
+testExecutionConcurrency=-1
+
+JunitMethodExecutionTimeout=5 m
```

Comment:
@akashkulk what's this for?

Reply:
In this PR, CDK 0.23.15 https://github.com/airbytehq/airbyte/pull/35827 default timeout was reduced from 5m to 1m by default. It wasn't really validated against any other certified connectors other than source-mssql for which it seems like this fix exclusively applied. 

This timeout just adds back that functionality.

Reply:
https://github.com/airbytehq/airbyte/pull/35827/files#diff-0cea538e87e1c9727a2504fe03af0371020c7b491d3824daa7e4559f02df1060